src/zmqports.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2014 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_ZMQPORTS_H
+#define BITCOIN_ZMQPORTS_H
+
+#if defined(HAVE_CONFIG_H)
+#include "config/bitcoin-config.h"
+#endif
+
+#include <string>
+
+// Global state
+extern bool fZMQPub;
+
+#if ENABLE_ZMQ
```

Comment:
Is the reason why the `#ifdefs` are here instead of wrapping the code-part in `init.cpp` to lower the #ifdef in `init.cpp`? Maybe also something for @jgarzik to answer.